templates/volunteer/email/internal_volunteer_profile_email_notification.txt

```diff
@@ -0,0 +1,28 @@
+{% extends "account/email/base_message.txt" %}
+{% load i18n %}
+{% load account %}
+{% block content %}
+Volunteer application for {{ profile.user.first_name }}
+{% if updated %}
+Your volunteer profile has recently been updated.
+{% endif %}
+
+Your current volunteer application status: {{ profile.application_status }}.
+
+Your information:
+- Your availability: {{ profile.availability_hours_per_week }} hours per week
+- GitHub username: {{ profile.github_username }}
+- Discord username: {{ profile.discord_username }}
+- Instagram username: {{ profile.instagram_username }}
+- Bluesky username: {{ profile.bluesky_username }}
+- Mastodon URL: {{ profile.mastodon_url }}
+- X username: {{ profile.x_username }}
+- LinkedIn URL: {{ profile.linkedin_url }}
+- PyLadies Chapter: {{ profile.pyladies_chapter }}
+- Region: {{ profile.region }}
+- Languages spoken: {{ profile.languages_spoken|join:", " }}
+- PyLadies Chapter: {{ profile.pyladies_chapter }}
+- Additional Comments: {{ profile.additional_comments }}
+
+If you would like to review or update your application at any time, go to your Volunteer Dashboard at https://{{ current_site.domain }}{% url 'volunteer:index' %}.
```

Comment:
I think this line can be removed for now. I think it should point to a "Review Volunteers" page that doesn't yet exist. We can update the email template later.

Reply:
resolved in [38b05a7](https://github.com/pyladies/pyladiescon-portal/pull/146/commits/38b05a7be95bd7b9220fa2efc5f9e502f28fed3d)